docs/nlp/nlp-extended-examples.mdx

```diff
@@ -740,7 +740,7 @@ Let's go through the examples.
 ### Answering Questions with Context
 
 <AccordionGroup>
-    <Accordion title="Example 1">
+    <Accordion title="Example Answering Questions with Context">
```

Comment:
Let's remove 'Example` from 'Example Answering Questions with Context' to keep it consistent with the above item.

Reply:
Solved, grate catches @martyna-mindsdb !